app/controllers/tokens_controller.rb

```diff
@@ -58,4 +60,9 @@ def ttl_seconds_in_hours
     hours = minutes.div(60)
     hours
   end
+
+  def throttle_limit_error
```

Comment:
I found this name a bit hard to parse. Might something like `show_throttle_limit_error` be clearer?